apm-protocol/apm-network/src/main/java/org/apache/skywalking/apm/network/trace/component/ComponentsDefine.java

```diff
@@ -143,4 +143,6 @@
     public static final OfficialComponent EHCACHE = new OfficialComponent(75, "Ehcache");
 
     public static final OfficialComponent SOCKET_IO = new OfficialComponent(76, "SocketIO");
+
+    public static final OfficialComponent REST_HIGH_LEVEL_CLIENT = new OfficialComponent(77, "rest-high-level-client");
```

Comment:
What is it? Why don't we use `TRANSPORT_CLIENT`?

Reply:
This is not recommended in the new ES version.

Reply:
Yes. HTTP Client is recommended.